precompiles/utils-macro/tests/02-parse-invalid-enum.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019-2021 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+/*#[precompile_utils_macro::generate_function_selector]
+pub enum Action {
+	Toto,
+	Tata,
+}*/
+
+fn main() {}
```

Comment:
Is this testing stuff done?

Reply:
In fact, the current CI does not allow me to test the error cases of my macro. Because `cargo test --all` imposes that all the code of all the tests compile well, but it is a compilation error that I want to test. I have to put a TODO on this test, the time to see if we modify the CI for that.

Reply:
Test deleted for now